mlflow/utils/credentials.py

```diff
@@ -76,6 +77,32 @@ def login(backend="databricks"):
         )
 
 
+def _is_valid_databricks_host(host):
+    # Check if the host is a valid Databricks URL: 1) starts with https://. 2) has "databricks.com"
+    # in the domain. 3) No params/query/fragment in the URL.
+    if not host.startswith("https://"):
+        _logger.error(f"Host must start with 'https://', but received: {host}")
+        return False
+
+    parsed_url = urllib.parse.urlparse(host)
+    if not "databricks.com" in parsed_url.netloc:
+        # A valid Databricks host must have "databricks.com" in the domain.
+        _logger.error(
+            f"Invalid Databricks host: {host}. The host must have 'databricks.com' in the domain."
+            "A valid URL is like https://community.cloud.databricks.com"
+        )
+        return False
```

Comment:
Not true for Azure Databricks. I don't think we should rely on this kind of validation. Can we remove it?